routes/truckRoutes.js

```diff
@@ -1,3 +1,88 @@
 // connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+var express = require('express');
+var trucks = require('../models/truckModel');

```

Comment:
This require call is returning a Truck model that you reference later as Truck, so the variable here should be Truck instead of trucks.